README.md

```diff
@@ -24,3 +24,13 @@ import { FacebookBoxIcon } from 'mdi-react';
 The icons get a class named `mdi-icon` attached for styling. You can also attach own additional classes with the `className` property.
 
 *Note that the second method from the example imports every icon from this package. Use the first method if your bundler does not support the removal of unused code.*
+
+
+## Typescript Definitions
+
+A typescript definition file will be created upon building.
+It will be located at:
+
+```
+/node_modules/mdi-react/build/typings.d.ts
```

Comment:
`build` is the root directory of the npm module. So it will be located in `node_modules/mdi-react/typings.d.ts`